test/bin/run-integration-tests.sh

```diff
@@ -48,6 +48,7 @@ PYTHONPATH="$( pwd )" \
     python test/bin/inject_api_key.py admin@user.com \
     "XZpXI40Uk85eozjQkU1zHJ6yZHpix+j0mo1TMeGZ4dPzIqVPVGPmyfeK"
 
+PYTHONPATH="$( pwd )" \
```

Comment:
We couldn't import the `api` package in the integ tests. I needed to in order to test that the log messages were logged properly.